static/js/components/tiles/bar_tile.tsx

```diff
@@ -283,21 +283,30 @@ function rawToChart(
   }
   // Optionally sort ascending/descending by value
   if (props.sort === "ascending" || props.sort === "descending") {
-    // sort variables in first group by value
-    dataGroups[0].value.sort(
-      (a, b) => (a.value - b.value) * (props.sort === "ascending" ? 1 : -1)
-    );
+    // if only one variable, sort by places
+    if (props.statVarSpec.length == 1) {
+      dataGroups.sort(
+        (a, b) =>
+          (a.value[0].value - b.value[0].value) *
```

Comment:
We're sure there will be one value in both a and b?  if not, add defensive check?

Reply:
Good call out! Added defensive check.